### PR TITLE
Allow multiple @Body objects in request methods

### PR DESCRIPTION
**Pull request by didalgolab**

### CODE REVIEW

**Review by sashirestela — COMMENTED** [review]
@didalgolab This is more an observation about styling: could you make explicit imports? If you think that is too effort, just comment on it and don't make changes.

**Comment by sashirestela** [comment]
@didalgolab I've just put all my observations. Please, let me know when you have fixed them.

**Comment by didalgolab** [comment]
> @didalgolab could you make explicit imports?

@sashirestela Fixed (it's automatic thing in my IDE, not intentional)

**Comment by sashirestela** [comment]
@didalgolab Are you aware of my three pending revisions about missing unit testing? (just to be sure)

**Comment by didalgolab** [comment]
Oh, you mean Codecov Report? I wasn't aware it's directed to me. Thx for heads up. I'm on it.

**Comment by sashirestela** [comment]
I meant this observations:
image

**Comment by didalgolab** [comment]
@sashirestela I cannot see comments that are "Pending". Please submit you review first.

**Review by sashirestela — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/sashirestela/cleverclient/util/JsonUtil.java [hunk 1]
```diff
@@ -1,50 +1,78 @@
...
+    public static String objectsToJson(List<?> objects) {
+        Map<String, Object> jsonProperties = new HashMap<>();
+        for (Object obj : objects) {
+            jsonProperties.putAll(objectMapperStrict.convertValue(obj, jsonPropertiesTypeRef));
+        }
+
+        return objectToJson(jsonProperties);
+    }
+
+    public static <T> Map<String, Object> objectToMap(T object) {
```
[the remark this answers is not in the document]
Comment (didalgolab) [reply src/main/java/io/github/sashirestela/cleverclient/util/JsonUtil.java:38]: Done.

File: src/main/java/io/github/sashirestela/cleverclient/util/JsonUtil.java [hunk 2]
```diff
@@ -1,50 +1,78 @@
...
         try {
-            return objectMapper.readValue(json, clazz);
-        } catch (JsonProcessingException e) {
+            return objectReaderIgnoringUnknown.readValue(json, clazz);
+        } catch (IOException e) {
+            throw new CleverClientException("Cannot convert the Json {0} to class {1}.", json, clazz.getName(), e);
+        }
+    }
+
+    public static <T> T jsonToObjectStrict(String json, Class<T> clazz) {
```
[the remark this answers is not in the document]
Comment (didalgolab) [reply src/main/java/io/github/sashirestela/cleverclient/util/JsonUtil.java:62]: Done.

File: src/main/java/io/github/sashirestela/cleverclient/util/ReflectUtil.java [hunk 3]
```diff
@@ -23,7 +24,20 @@ public static <T> T createProxy(Class<T> interfaceClass, InvocationHandler handl
...
+        Map<String, Object> structure = new HashMap<>();
+        for (Object object : objects) {
+            structure.putAll(getMapFields(object));
+        }
+
+        return structure;
+    }
+
     public static Map<String, Object> getMapFields(Object object) {
+        if (object instanceof Map) {
```
[the remark this answers is not in the document]
Comment (didalgolab) [reply src/main/java/io/github/sashirestela/cleverclient/util/ReflectUtil.java:37]: Done.

**Review by sashirestela — APPROVED** [review]

**Pull request merged**